packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -275,6 +260,60 @@ export default {
          * focus on the input field.
          */
 
+        describeFirstnameErrorMessage () {
+            if (this.$v.firstName.$dirty) {
+                if (!this.$v.firstName.required) {
+                    return 'Please include your first name';
```

Comment:
Perhaps one for later (Worth spinning up a ticket) to make sure we localise these messages